src/lib/fragments/CookiePages/CookiesPage.js

```diff
@@ -0,0 +1,184 @@
+import React, { useEffect, useState } from 'react';
+
+import { Article, Heading } from './Cookies.styles';
+
+import { handleCookieAccept } from '../../components/cookies/utils/Cookie';
+
+import Cookies from "js-cookie";
+
+
+
+const SuccessNotification = (props) => {
+  return (
+    <div className="govuk-notification-banner govuk-notification-banner--success govuk-!-margin-top-1"
+         role="alert" aria-labelledby="govuk-notification-banner-title"
+         data-module="govuk-notification-banner">
+      <div className="govuk-notification-banner__header">
+        <h2 className="govuk-notification-banner__title" id="govuk-notification-banner-title">
+          Success
+        </h2>
+      </div>
+      <div className="govuk-notification-banner__content">
+        <p className="govuk-notification-banner__heading">
+          You’ve updated your cookie preferences.
+        </p>
+      </div>
+    </div>
+  );
+};
+
+const CookiesPage = ({ ...props }) => {
+    const [cookieState, setCookieState] = useState(null);
+    const [updateState, setUpdateState] = useState(false);
+    const { tag } = props;
```

Comment:
Would it be worth using context for this and the dispatch.  Rather than reimplmenting this. Or just custom hook with its own useEffect in it